src/main/java/build/buildfarm/worker/ExecuteActionStage.java

```diff
@@ -103,13 +105,17 @@ synchronized int removeAndRelease(String operationName, int claims) {
   }
 
   public void releaseExecutor(
-      String operationName, int claims, long usecs, long stallUSecs, int exitCode) {
+      String operationName,
+      int claims,
+      @s(Prefix.micro) long uSecs,
+      @s(Prefix.micro) long stallUSecs,
+      int exitCode) {
     int slotUsage = removeAndRelease(operationName, claims);
-    executionTime.observe(usecs / 1000.0);
-    executionStallTime.observe(stallUSecs / 1000.0);
+    executionTime.observe(TimeUnit.MILLISECONDS.convert(uSecs, TimeUnit.MICROSECONDS));
+    executionStallTime.observe(TimeUnit.MILLISECONDS.convert(stallUSecs, TimeUnit.MICROSECONDS));
```

Comment:
This wasn't part of the CheckerFramework, but I just didn't trust myself to do unit conversions :) Can revert..